app/controllers/report_controller/reports/editor.rb

```diff
@@ -1,5 +1,6 @@
 module ReportController::Reports::Editor
   extend ActiveSupport::Concern
+  include ReportHelper
```

Comment:
..And this include should not be needed at all (still a part of ReportController).